src/cartesian/ErrorBar.tsx

```diff
@@ -48,8 +48,20 @@ interface ErrorBarProps extends InternalErrorBarProps {
 export type Props = SVGProps<SVGLineElement> & ErrorBarProps;
 
 export function ErrorBar(props: Props) {
-  const { offset, layout, width, dataKey, data, dataPointFormatter, xAxis, yAxis, ...others } = props;
-  const svgProps = filterProps(others, false);
+  const {
+    offset = 0,
+    layout = 'horizontal',
+    width = 5,
```

Comment:
Visual regression https://www.chromatic.com/test?appId=63da8268a0da9970db6992aa&id=664cfc9eadd585053f5b885c

This will need to be converted to a class component unfortunately :/

Reply:
Actually... it works in 2.x but not 3.x. Not sure why, we haven't touched error bar much. @PavelVanecek ?

Reply:
I'll just make it a class component to be safe.

Reply:
Thanks